website/dbt-versions.js

```diff
@@ -1,4 +1,9 @@
 exports.versions = [
+  {
+    version: "1.5 (beta)",
```

Comment:
@matthewshaver Do you know how this works with version blocks, the URL, any other pieces of logic we might have that expect the value to be numeric?

Reply:
That's a good point and I haven't versionblocked anything to test. Let me run through a few scenarios and talk to the frontend team about it.

Reply:
@jtcohen6 Going to pause the merging of the framework without the (beta) tag. Asana task opened and should have this next week. Would like to have a uniform experience from the start, but if you'd rather move forward ASAP let me know.

Reply:
@matthewshaver I disagree - I'd rather get this live sooner!

I'm totally on board with opening a task for creating a visual distinction (i.e. the word "pre" or "prerelease") in the version dropdown when a version has `isPrerelease: true`. This should be _visual only_, with no change to the version value, and no need to do anything differently with `VersionBlock` components.

I don't believe that UI work should be a blocker for initializing the v1.5 docs. It would be great to have when we have it, but not a prerequisite, and not too disorienting for someone to see the change partway through.

Meantime, someone will still see this banner at the top of every page, when they select a prerelease version:

<img width="807" alt="Screenshot 2023-02-19 at 17 40 46" src="https://user-images.githubusercontent.com/13897643/219961864-a25fd4e9-55cb-474a-9240-382e438a3251.png">

Reply:
@jtcohen6 Fair point! Going through the editorial/style review now. Will publish as soon as I'm done,